### ISSUE TITLE
PW intake: <Add Spanish toggle and set up left nav> from <Sitewide CAIA>

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#70250]
### Indicate if you need front-end engineering support for

- [ ] Content audit
- [ ] Implementation of REACT sign-in widgets
- [ ] Support for UAT testing or phased roll out of app/tool landing page
- [ ] Implementation of Translation widgets
- [X] Something else

### Describe your request

We've published 2 pages in Spanish that we need to set up the left nav for and add the Spanish toggle at the top of the page. 

These pages are:

- https://www.va.gov/decision-reviews/supplemental-claim-esp/
- https://www.va.gov/report-harassment-esp/
- 

### Time constraints / launch deadlines

no hard timing

### Your product/team Slack channel

Sitewide CAIA

### Your OCTODE product owner

Randi Hecht

### Notify Sitewide Public Websites of this issue, and tag @Fran Cross & @jilladams.

- [X] I acknowledge that I must notify `#sitewide-public-websites` in Slack after submitting this issue.

**Comment by FranECross** [comment]
@randimays @chriskim2311 @dsasser Does this request need a Drupal as well as an FE ticket? Thanks in advance!

**Comment by randimays** [comment]
@FranECross I don't think we need FE changes, but I can't say that with 100% certainty. At minimum, I think we need to make the Drupal changes and see how things render on the FE first.

- `/decision-reviews/supplemental-claim-esp`: we need to add the Spanish link/translation to `File a Supplemental Claim` in the `Decision reviews` menu in Drupal. This page is using the same template as another set with language toggles ([/health-care-how-to-apply](https://www.va.gov/health-care/how-to-apply/))
- `/report-harassment-esp`: this one's English page doesn't have a sidebar at all. I assume we need to add a menu in Drupal and assign it to this page. It's also using the same template as the working set with language toggles.

**Comment by FranECross** [comment]
@randimays Will you please take a look at Randi Hecht's comment above and let me know if her info/response unblocks this ticket? Thanks!

**Comment by randimays** [comment]
@FranECross I believe this is the ticket we discussed in planning last week. We decided that [#15759](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15759) may pave the way to fixing this issue, so we were going to wait until that ticket was resolved to try to tackle this one.

**Comment by FranECross** [comment]
The ticket formerly blocking this one has been completed, paving the way for this to be worked on

**Comment by RLHecht** [comment]
@FranECross would your team be able to prioritize the more recent toggle ticket I opened yesterday over this one? https://github.com/department-of-veterans-affairs/va.gov-team/issues/71640 We'd love to get this one live before the end of the year if at all possible. Thank you!

**Comment by jilladams** [comment]
I went ahead and updated #16392 with that in mind. Thanks for flagging, @RLHecht
